mars/dataframe/contrib/raydataset/dataset.py

```diff
@@ -16,10 +16,10 @@
 from .mldataset import _rechunk_if_needed
 from typing import Dict, List
 
-ray = lazy_import("ray")
+ray = lazy_import("ray", globals=globals())
```

Comment:
Can we just using `globals()` in `lazy_import` if not passed? If users don't want to update global namespace, they can pass ean emtpy dict

Reply:
Done for that.